### DISCUSSION TITLE
Icon Update (MacOS)

### DISCUSSION THREAD

**Discussion by Mykbibby** [neanes/neanes#384] [Ideas]
Hey there -- I took the Neanes icon you designed and applied Apple's design guidelines post Big Sur. Feel free to use it if you'd like. The current version was standing out on my dock so I thought I'd quickly tweak it and share it. Happy to refine it or provide iOS/iPad OS versions if you'd like. Thanks again for everything... this app is a literal godsend. 

<img width="485" alt="Screenshot 2023-10-31 at 8 25 30 PM" src="https://github.com/danielgarthur/neanes/assets/44008080/fba2e45b-aea7-4c90-a8fd-a6268b32c289">
<img width="485" alt="Screenshot 2023-10-31 at 8 35 40 PM" src="https://github.com/danielgarthur/neanes/assets/44008080/7d22032c-11bb-4eb3-88b1-b891e86fd270">

**[Neanes Icon (mac).icns.zip](https://github.com/danielgarthur/neanes/files/13223152/Neanes.Icon.mac.icns.zip)**

**Comment by danielgarthur** [comment]
Thanks for the icons. These have been included in `v0.4.6`.
